### DISCUSSION TITLE
使用 Micopython 开发DVI

### DISCUSSION THREAD

**Discussion by Carlos-TGW** [micropython/micropython#15224] [RP2040 / Pico]
Does anyone have experience developing DVI with Micopython

**Comment by bikeNomad** [comment]
Scott Shawcroft did a DVI output in CircuitPython using the RP2040 PIO: https://www.adafruit.com/product/5710

**Reply by Carlos-TGW** [reply]
Thank you for your reply, but I want to use Micopython platform for development, I wonder if you have relevant experience. As far as I know, PicoDVI of CircuitPython does not seem to support Micopython
